make/RunTests.gmk

```diff
@@ -856,6 +856,11 @@ define SetupRunJtregTestBody
     JTREG_AUTO_PROBLEM_LISTS += ProblemList-zgc.txt
   endif
 
+  ifneq ($$(DEBUG_LEVEL), release)
+    JTREG_AUTO_PROBLEM_LISTS += ProblemList-debug.txt
+    JTREG_AUTO_TIMEOUT_FACTOR := 10
```

Comment:
Please don't do this. We set timeout factors as determined by how the tests will be run in our CI and this could interfere with that.

Reply:
The priority of TIMEOUT_FACTOR set outside is higher than in this makefile. This is set the default value. seems like to https://github.com/openjdk/jdk/blob/master/make/RunTests.gmk#L848 or https://github.com/openjdk/jdk/blob/master/make/RunTests.gmk#L852